src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -101,6 +102,31 @@ default ThreadMember getThreadMemberById(String id)
     @Nullable
     ThreadMember getThreadMemberById(long id);
 
+    @NotNull
+    @CheckReturnValue
+    default RestAction<ThreadMember> retrieveThreadMember(Member member)
+    {
+        return retrieveThreadMemberById(member.getId());
```

Comment:
Null check

Reply:
You are currently doing `Long.parseLong(Long.toString(getIdLong())` with this code

Reply:
Replaced to getIdLong on both occasions, and added a Checks#notNull